CONTRIBUTING.md

```diff
@@ -0,0 +1,38 @@
+# Contributing to Gravwell's Open Source Repo
+
+We welcome contributions in the form of issues and Github pull requests. This file documents some guidelines to keep in mind when contributing to the Gravwell repo.
+
+## Content
+
+Good issues should contain a detailed description of your problem and either steps to reproduce it or a minimal reproducer program.
+
+Good pull requests should address problems which may be applicable to others, not just your specific use-case.
+
+Please don't submit PRs which only make small modifications to comments or documentation; we are not a resume-padding service.
```

Comment:
Not sure that "we are not a resume-padding service" is really necessary here, but I won't object if you want to keep it.